perfdash/perfdash.go

```diff
@@ -81,12 +86,19 @@ func run() error {
 		options.DefaultBuildsCount = maxBuilds
 	}
 
-	gcs, err := NewGCSMetricsBucket(*logsBucket, *logsPath, *credentialPath)
+	var metricsBkt MetricsBucket
+	var err error
+
+	if options.Mode == s3Mode {
+		metricsBkt, err = NewS3MetricsBucket(*logsBucket, *logsPath, *awsRegion)
+	} else {
+		metricsBkt, err = NewGCSMetricsBucket(*logsBucket, *logsPath, *credentialPath)
+	}
```

Comment:
Since you touch this code - let's not panic and simply return an error below.

Reply:
That's a good point. Done.